libmultilabel/model.py

```diff
@@ -59,40 +60,55 @@ def training_step(self, batch, batch_idx):
         return loss
 
     def validation_step(self, batch, batch_idx):
-        loss, pred_logits = self.shared_step(batch)
-        return {'loss': loss.item(),
-                'pred_scores': torch.sigmoid(pred_logits).detach().cpu().numpy(),
-                'target': batch['label'].detach().cpu().numpy()}
+        self._shared_eval_step(batch, datch_idx)
+
+    def validation_step_end(self, batch_parts):
+        self._shared_eval_step_end(batch_parts)
 
     def validation_epoch_end(self, step_outputs):
-        eval_metric = self.evaluate(step_outputs, 'val')
-        return eval_metric
+        return self._shared_eval_epoch_end(step_outputs, 'val')
 
     def test_step(self, batch, batch_idx):
-        return self.validation_step(batch, batch_idx)
+        return self._shared_eval_step(batch, batch_idx)
+
+    def test_step_end(self, batch_parts):
+        self._shared_eval_step_end(batch_parts)
```

Comment:
```suggestion
        return self._shared_eval_step_end(batch_parts)
```
Since they are aliases, you probably want them to be exactly the same. Works with `return None` too.

Reply:
You're right. Thanks.